modules/casper.js

```diff
@@ -592,23 +661,67 @@ Casper.prototype.die = function die(message, status) {
  * @param  String  targetPath  The destination file path
  * @param  String  method      The HTTP method to use (default: GET)
  * @param  String  data        Optional data to pass performing the request
+ * @param  Boolean async       Asynchroneous request? (default: false)
  * @return Casper
  */
-Casper.prototype.download = function download(url, targetPath, method, data) {
+Casper.prototype.download = function download(url, targetPath, method, data, async) {
     "use strict";
     this.checkStarted();
-    var cu = require('clientutils').create(utils.mergeObjects({}, this.options));
-    try {
-        fs.write(targetPath, cu.decode(this.base64encode(url, method, data)), 'wb');
+    async = ((typeof(async) !== "undefined")? async : false);
+	if ((phantom.version.major > 1) && (!async)) {
```

Comment:
Same here: let's tell the users "don't do this, use the way (async) that will be supported down the road". :)